src/promptflow/promptflow/contracts/multimedia.py

```diff
@@ -147,9 +147,11 @@ def _from_url(url: str, mime_type: str = None):
             )
 
     @staticmethod
-    def _from_dict(image_dict: dict):
+    def _from_dict(image_dict: dict, base_dir: Path = None):
```

Comment:
why do we need `base_dir` here?
We should be loading from either absolute path or relative path to current working dir.
In other words, the path should be always readable without modifying.

Reply:
I will change working dir in test case and wil remove base_dir